hack/util.sh

```diff
@@ -388,8 +435,10 @@ reset_tmp_dir() {
 function kill_all_processes()
 {
 	sudo=
-	if type sudo &> /dev/null; then
-		sudo=sudo
+	if [ -n "$SUDO" ]; then
+		if type sudo &> /dev/null; then
+			sudo=sudo
```

Comment:
> Shouldn't SUDO be set as well ?

`$SUDO` is just a marker for whether you want to run the commands as root.  Its value doesn't matter.